silk/tests/test_silk.py

```diff
@@ -33,3 +34,9 @@ def test_error_msg_response(dd_run_check, aggregator, instance):
         dd_run_check(check)
         aggregator.assert_service_check('silk.can_connect', SilkCheck.OK)
         aggregator.assert_service_check('silk.state', SilkCheck.WARNING)
+
+
+def test_events():
+    normalized_event = SilkEvent(SAMPLE_RAW_EVENT, ["test:silk"])
+    actual_payload = normalized_event.get_datadog_payload()
+    assert actual_payload == EXPECTED_EVENT_PAYLOAD
```

Comment:
move to test_events.py. We should add a unit test where we override the `self.latest_event_query` to test the # of submitted metrics that we expect

```suggestion
def test_event_payload():
    normalized_event = SilkEvent(SAMPLE_RAW_EVENT, ["test:silk"])
    actual_payload = normalized_event.get_datadog_payload()
    assert actual_payload == EXPECTED_EVENT_PAYLOAD
```